server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/UploadContract.scala

```diff
@@ -75,7 +75,7 @@ trait UploadContract {
       .basePath("")
       .header(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER)
     .when
-      .get(s"/download/$ACCOUNT_ID/$blobId")
+      .get(s"/download/$ACCOUNT_ID$blobId")
```

Comment:
is it a mistake?

Reply:
Likely to be with the 404 errors I can see from the build